context/AuthContextProvider.test.tsx

```diff
@@ -32,21 +32,29 @@ describe('AuthContextProvider', () => {
     password: 'password1234',
   });
 
+  const setIsSignedIn = jest.fn();
+  const resetUser = jest.fn();
+
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
   test('should show success notification after a successful login', async () => {
-
     mockCreateEmailPasswordSession.mockResolvedValue({});
 
-    await loginAccount({user, router, getUser});
+    await loginAccount({ user, router, getUser });
```

Comment:
Some of tests seem to be failing on build. Please collaborate with the team to make sure these pass.